src/interaction/VerletListVSphereInteractionTemplate.hpp

```diff
@@ -127,9 +127,7 @@ namespace espressopp {
         if(potential._computeForce(force, fsi, fsj, p1, p2)) {
           p1.force() += force;
           p2.force() -= force;
-          p1.fradius() = fsi;
-          p2.fradius() = fsj;
```

Comment:
Those two properties are used in integrator/VelocityVerletOnRadius.cpp;
Could you please comment why you do not compute and assign them anymore?